ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -419,6 +427,26 @@ public static CCDCase getCCDCitizenCase(List<CCDCollectionElement<CCDAmountRow>>
             .paymentReference("RC-1524-6488-1670-7520")
             .applicants(applicants)
             .respondents(respondents)
+            .directionList(ImmutableList.of(
```

Comment:
should we have this in different method, so that existing flow is not disturbed? As well as do we get complete details of respondent for order?

Reply:
Sure, I'll create a new method 👍